decidim-core/app/controllers/concerns/decidim/use_organization_time_zone.rb

```diff
@@ -25,7 +25,7 @@ def use_organization_time_zone(&action)
       #
       # Returns a String.
       def organization_time_zone
-        @organization_time_zone ||= current_organization.time_zone
+        @organization_time_zone ||= current_organization.time_zone ||= current_user.time_zone
```

Comment:
```suggestion
        @organization_time_zone ||= current_user.time_zone || current_organization.time_zone
```
we want the assignation of this to check the user first, then the organization.
Also: carefull to do double assignations with the `||=` statement.